tests/test_api.py

```diff
@@ -0,0 +1,94 @@
+import os.path, sys, uuid
+
+from pytest import fixture, mark
+from kubernetes import client, config as k8s_config, utils
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+try:
+    from app import app
+except Exception as e:
+    pass
```

Comment:
This is needed or unit tests in Docker will complain there is no kubeconfig. Probably `app.py` should be reworked so that nothing actually happens on import?